addons/mail/static/src/components/thread_view/thread_view.xml

```diff
@@ -39,6 +39,12 @@
                         <t t-elif="threadView.composerView">
                             <div class="o-autogrow"/>
                         </t>
+                        <t t-if="threadView.hasReactionsSummary">
+                            <MessageReactionsSummary
+                                messageReactionGroupLocalId="threadView.messageReactionGroup.localId"
+                                localId="threadView.messageReactionGroup.message.messageReactionsSummary.localId"
+                            />
+                        </t>
```

Comment:
pattern to follow:

```
<t t-if="threadView.messageReactionsSummary">
    <MessageReactionsSummary messageReactionsSummaryLocalId="threadView.messageReactionsSummary.localId"/>
</t>
```

Reply:
Same remark about the the need of `messageReactionGroupLocalId` to highlight the corresponding reaction user has clicked to open the summary